src/main/java/tahub/contacts/model/ModelManager.java

```diff
@@ -195,13 +194,13 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
 
     @Override
     public Path getScaListFilePath() {
-        return userPrefs.getCourseListFilePath();
+        return userPrefs.getScaListFilePath();
     }
 
     @Override
-    public void setScaListFilePath(Path courseListFilePath) {
-        requireNonNull(courseListFilePath);
-        userPrefs.setCourseListFilePath(courseListFilePath);
+    public void setScaListFilePath(Path scaListFilePath) {
+        requireNonNull(scaListFilePath);
+        userPrefs.setScaListFilePath(scaListFilePath);
     }
 
```

Comment:
@miloaisdino probably copied wrongly last time, doesnt seem to impact functionality though